app/assets/javascripts/application.js

```diff
@@ -1144,3 +1144,29 @@ function appendOptionsToDropdown(options, selectElement) {
     });
     selectElement.append(optionsArray);
 }
+
+// dynamically disable click events on DOM elements using jQuery selector
+// will not allow pointer events on mouse, and will grey out element (50% opacity)
```

Comment:
Using `/** comment */` style enables editors to show function docs upon hover.
```suggestion
// dynamically disable click events on DOM elements using jQuery selector
// will not allow pointer events on mouse, and will grey out element (50% opacity)
```

```suggestion
/** 
* Dynamically disable click events on DOM elements using jQuery selector
* will not allow pointer events on mouse, and will grey out element (50% opacity)
*/
```

Reply:
27de521